modules/nw-ovn-kubernetes-list-resources.adoc

```diff
@@ -25,45 +25,43 @@ $ oc get all,ep,cm -n openshift-ovn-kubernetes
 .Example output
 [source,terminal]
 ----
-NAME                       READY   STATUS    RESTARTS      AGE
-pod/ovnkube-master-9g7zt   6/6     Running   1 (48m ago)   57m
-pod/ovnkube-master-lqs4v   6/6     Running   0             57m
-pod/ovnkube-master-vxhtq   6/6     Running   0             57m
-pod/ovnkube-node-9k9kc     5/5     Running   0             57m
-pod/ovnkube-node-jg52r     5/5     Running   0             51m
-pod/ovnkube-node-k8wf7     5/5     Running   0             57m
-pod/ovnkube-node-tlwk6     5/5     Running   0             47m
-pod/ovnkube-node-xsvnk     5/5     Running   0             57m
+NAME                       READY   STATUS    RESTARTS        AGE
+pod/ovnkube-master-2qnbv   6/6     Running   0               19m
+pod/ovnkube-master-kqmnv   6/6     Running   1 (11m ago)     19m
```

Comment:
why are these bits changing? we are showing an output with more restarts :P just make them 0 restarts or do we really care about pod names on docs?